app/controllers/audits/base_controller.rb

```diff
@@ -13,12 +23,16 @@ def filter(override = {})
         primary_org_only: primary_org_only?,
         sort_by: params[:sort_by],
         title: params[:query],
-      }.merge(override)
+      }
 
-      Filter.new(options)
+      options.delete_if do |_, v|
+        v == "" || v.nil?
+      end
     end
 
-  private
+    def default_filter
```

Comment:
Using a template method here is very handy. But default_filter is only used in the index action. Can you think of a way to make it more explicit?